### DISCUSSION TITLE
Proposal: total beatmaps and collection numbers to be added in laser

### DISCUSSION THREAD

**Discussion by gamedestroyer222** [ppy/osu#22930] [Ideas] [answered]
Just wondering but is the total beatmaps number and the number for how many maps you have in a certain collection planned to be implemented in laser? Since I like to watch those numbers go up and down in stable and I'm pretty sure others do aswell

**Comment by peppy** [comment, marked as answer]
In the next release you will be able to see a count at song select, which should mostly resolve this for you:

![osu! 2023-03-22 at 04 30 38](https://user-images.githubusercontent.com/191335/226802720-4cadb08f-0c45-4434-a89f-cf71e48de476.png)

**Reply by peppy** [reply]
I've also added counts to the manage dialog (#22932).

**Reply by gamedestroyer222** [reply]
Thanks peppy this should help the players that like to use their collections to went to play laser more I know it has for me
